tests/unit/utils/test_version.py

```diff
@@ -894,7 +894,8 @@ class VersionParams:
     VersionParams('unknown-dist', known_distribution=False),
     VersionParams('no-ssl', ssl_support=False),
 ], ids=lambda param: param.name)
-def test_version_output(params, stubs, monkeypatch):
+@pytest.mark.parametrize('autoconfig_loaded', [True, False])
```

Comment:
This runs all `test_version_output` tests twice (with autoconfig and without). Instead, please add another `autoconfig_loaded` attribute (probably True by default) to `VersionParams` above, and then add a single test with doesn't have autoconfig loaded (like the tests above this line).